build_config.rb

```diff
@@ -26,6 +26,7 @@
 
   # ngx_mruby extended class
   conf.gem './mrbgems/ngx_mruby_mrblib'
+  conf.gem :github => 'ksss/mruby-stringio' # required by rack-based-api
```

Comment:
Could you add to https://github.com/matsumoto-r/ngx_mruby/blob/master/mrbgems/rack-based-api/mrbgem.rake as rack-based-api dependency?